src/OpenFeature.Contrib.Providers.Flagsmith/OpenFeature.Contrib.Providers.Flagsmith.csproj

```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <TargetFrameworks>netstandard20</TargetFrameworks>
     <PackageId>OpenFeature.Contrib.Providers.Flagsmith</PackageId>
-    <VersionNumber>0.1.0</VersionNumber>
+    <VersionNumber>0.1.1</VersionNumber>
```

Comment:
This doesn't need to be manually adjusted.

Reply:
done